Patches/Epona Race/Ammo/Ammo_Epona.xml

```diff
@@ -164,6 +164,80 @@
 				</value>
 			</li>
 			
+			<!-- ==================== Flamer Variant ========================== -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_NuckelaveeShotgun</defName>

```

Comment:
I get errors when attempting to fire the Nuckelavee Shotgun. It appears that the Zebra Shotgun has been made compatible, but the Nuckelavee one hasn't? It appears to still have the vanilla percentile accuracy stats.